add_new_dc_test.py

```diff
@@ -90,7 +90,7 @@ def add_node_in_new_dc(self) -> BaseNode:
         self.monitors.reconfigure_scylla_monitoring()
 
         status = self.db_cluster.get_nodetool_status()
-        assert len(status.keys()) == 2, f"new datacenter was not registered. Cluster status: {status}"
+        assert len(status.keys()) == 2, f"new datacenter was not registered. Cluster status: {status}"  # noqa: PLR2004
```

Comment:
According to our situation in SCT, we may want to disable "PLR2004" rule in config.

Reply:
my idea was to skip it for current code, and make sure it's enable from now on..

but yes, this generate a big amount of changes

Reply:
I've ignored it in the configuration for now